terms.html

```diff
@@ -91,12 +100,12 @@
 specifications. This specification decouples the <a>identity provider</a>
 concept into two distinct concepts: the <a>issuer</a> and the <a>holder</a>.
   </dd>
-  <dt><dfn data-lt="issuers|issuer's">issuer</dfn></dt>
+  <dt><dfn data-lt="issuers|issuer's|issuer(s)|issuing|issued">issuer</dfn></dt>
   <dd>
-A role an <a>entity</a> can perform by asserting <a>claims</a> about one or
-more <a>subjects</a>, creating a <a>verifiable credential</a> from these
-<a>claims</a>, and transmitting the <a>verifiable credential</a> to a
-<a>holder</a>.
+A role an <a>entity</a> can perform as it constructs a <a>verifiable credential</a> 
+from a set of <a>claims</a> it has authored, including metadata and proofs,
```

Comment:
See comment on authorship. I do not believe authoring the claims is part of how VCs work.
```suggestion
from a set of <a>claims</a> it asserts as true, 
```